requirements.txt

```diff
@@ -0,0 +1,7 @@
+opencv-python==3.3.0.10
+tqdm==4.19.4
+numpy==1.13.3
+imgaug==0.2.5
+tensorflow==1.4.0
+h5py==2.7.1
+
```

Comment:
Get rid of this extra newline then good to go

Reply:
Oh, sorry! Thought that @SUPERCILEX wanted that.

Reply:
My B for not being clear: you should have 1 line at the end of a file, but not 2 (you have 2 here).